nservicebus/gateway/index.md

```diff
@@ -97,3 +97,12 @@ To disable retries in the gateway use the `DisableRetries` setting:
 snippet: GatewayDisableRetriesConfiguration
 
 When retries are disabled, any messages that fail to be sent to another site will be immediately routed to the configured error queue.
+
+
+### Version compatibility
+
+The Gateway component ensures only forward compatibility for one major version. So a message sent by the NServiceBus 3.x Gateway can be understood by the NServiceBus 4.x Gateway, and a message sent by the NServiceBus 4.x Gateway can be understood by NServiceBus.Gateway 1.x (the independent package targeting NServiceBus 5.x).
+
+However, a message sent by the NServiceBus 3.x Gateway will not be understood by NServiceBus.Gateway 1.x (NServiceBus 5.x) as this skips a major version. Likewise, a message sent by NServiceBus.Gateway 2.x will not be understood by NServiceBus.Gateway 1.x, as backwards communication is not supported.
+
+Since a Gateway on the receiving end exists to receive HTTP requests and translate them to messages on the queue, it's possible to deploy multiple Gateway instances using different major versions, either on different servers, or using different ports on the same server. As long as these Gateway instances contain the same message routing settings, they don't require any handler logic. This can be useful during a gradual, endpoint-by-endpoint upgrade to ensure that each deployed endpoint can communicate with a remote Gateway instance that is compatible with it.
```

Comment:
>As long as these Gateway instances contain the same message routing settings, they don't require any handler logic. 

Is this true? The GW always delivers to its "local" endpoint if I'm not mistaken?

https://github.com/Particular/NServiceBus.Gateway/blob/develop/src/NServiceBus.Gateway/Routing/Endpoints/EndpointRouter.cs#L11 

so I think the only option is to upgrade the GW enpoints "major to major" which shouldn't be a huge deal since gateways communicate within the serivice boundary so it shouldn't cause ripple effects.